### PR TITLE
PLAT-632 Package Openhim Mapping Mediator

### PR DESCRIPTION
**Pull request by nour-borgi**
Packaging of: https://jembi.github.io/openhim-mediator-mapping/docs/gettingStarted/introduction

### CODE REVIEW

File: openhim-mapping-mediator/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '3.9'
+
+services:
+  openhim-mapping-mediator:
+    image: jembi/openhim-mediator-mapping
+    environment:
+      OPENHIM_REGISTER: "true"
```
Comment (michaelloosen) [inline openhim-mapping-mediator/docker-compose.yml:7]: These should all be variables that default to this value in the package metadata file, please.

File: openhim-mapping-mediator/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: '3.9'
+
+services:
+  openhim-mapping-mediator:
+    image: jembi/openhim-mediator-mapping
```
Comment (michaelloosen) [inline openhim-mapping-mediator/docker-compose.yml:5]:
```suggestion
    image: jembi/openhim-mediator-mapping:v3.0.0
```

**Review by michaelloosen — CHANGES_REQUESTED** [review]
Looking good, with minor fixes.

File: openhim-mapping-mediator/docker-compose.yml (hunk 2 above)
Comment (nour-borgi) [reply openhim-mapping-mediator/docker-compose.yml]: cool fixed in [695643e](https://github.com/jembi/platform/pull/193/commits/695643ea720c65f3d5bb34d7cf4e430518dbaf71)

File: openhim-mapping-mediator/docker-compose.yml (hunk 1 above)
Comment (nour-borgi) [reply openhim-mapping-mediator/docker-compose.yml]: cool fixed in [695643e](https://github.com/jembi/platform/pull/193/commits/695643ea720c65f3d5bb34d7cf4e430518dbaf71)

**Review by bradsawadye — APPROVED** [review]

**Review by michaelloosen — APPROVED** [review]
LGTM

**Pull request merged**